src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1555,43 +1581,45 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </PropertyGroup>
 
     <MSBuild
-        Projects="%(_MSBuildProjectReferenceExistent.Identity)"
-        Targets="GetTargetFrameworkProperties"
+        Projects="@(_MSBuildProjectReferenceCrossTargeting)"
```

Comment:
Could non cross-targetting projects ever have special props for their target framework?